lib/ansible/modules/network/edgeos/edgeos_command.py

```diff
@@ -149,7 +153,7 @@ def main():
     try:
         conditionals = [Conditional(c) for c in wait_for]
     except AttributeError as e:
-        module.fail_json(msg=str(e))
+        module.fail_json(msg=to_native(e))
```

Comment:
you want to use `to_text` instead of `to_native`

Reply:
One day I won't get those mixed up.